CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_salesforce v1.0.2
+[PR #52](https://github.com/fivetran/dbt_salesforce/pull/52) includes the following updates:
+
+## Bug fixes
+- Updated model `int_salesforce__date_spine` to accommodate when model `salesforce__lead` exists but has no records. In this case, the model now defaults to a range of one-month from the current date.
```

Comment:
@fivetran-joemarkiewicz Ahh you're right. Thank you! I had meant to reference the staging model, but that's not even right since the datespine looks at the source. Updated!